torch/_inductor/decomposition.py

```diff
@@ -447,11 +447,15 @@ def view_copy_dtype(self, dtype):
 
 @register_extra_random_decomp([aten.bernoulli_])
 def bernoulli_(self, p=0.5):
+    if self.device == torch.device("cpu"):
```

Comment:
I don't believe this will be properly marked as a mutation op, so there could be a correctness issue.

Reply:
Sorry, I did not get your point. As for bernoulli_, there is a ut that is passed when fallback_random=True, and the changes here have the same effect as fallback_random=True. Inplace bernoulli on CPU will be handled by [bernoulli_ in lowering](https://github.com/pytorch/pytorch/pull/97002/files#diff-a1b077971cddfabfa0071c5162265066e867bc07721816d95b9cbe58431c38e3L1059-L1067), which can handle mutation properly.https://github.com/pytorch/pytorch/blob/b507d7d798a17d6297871545db0498f6d1d9af07/test/inductor/test_torchinductor.py#L4531-L4542